package.json

```diff
@@ -19,5 +19,8 @@
   "bugs": {
     "url": "https://github.com/tiredsoldier/orula/issues"
   },
-  "homepage": "https://github.com/tiredsoldier/orula#readme"
+  "homepage": "https://github.com/tiredsoldier/orula#readme",
+  "dependencies": {
```

Comment:
no need to add this to root package.json - it is already in `server` package.json (we're just using the top level for convenience)